docs/v2-faq.md

```diff
@@ -29,10 +188,11 @@ For example, when your addon exposes a few components at the root level of `src/
 addon.publicEntrypoints('index.js', 'components/*.js'),
 ```
 
-Additionally, there is a feature supported in node.js and modern bundlers to define an `exports` key in your `package.json` with a mapping of export paths to the actual files on disk, that lets you further tweak or constrain your public exports. This is explained in more detail [here](https://nodejs.org/api/packages.html#exports).
+Additionally, there is a feature supported in node.js and modern bundlers to define an `exports` key in your `package.json` with a mapping of export paths to the actual files on disk, that lets you further tweak or constrain your public exports. This is explained in more detail here:
+- https://dev.to/binjospookie/exports-in-package-json-1fl2
```

Comment:
I don't feel right linking to `random` dev.to blog posts that can change on a whim their content
with the other two at least they are an org that gives us some assurance of the content being stable

Reply:
Removed this link. This was from the original [AUTHORING_LIBRARIES.md](https://github.com/embroider-build/addon-blueprint/blob/main/AUTHORING_LIBRARIES.md) file. However, I am not sure I agree with your reasoning, but I think the other links are sufficient and more authoritative...